qemu/tests/win_virtio_driver_update_test.py

```diff
@@ -39,13 +39,7 @@ def change_virtio_media(cdrom_virtio):
         logging.info("Changing virtio iso image to '%s'" % virtio_iso)
         vm.change_media("drive_virtio", virtio_iso)
 
-    if params.get("need_uninstall") == "yes":
-        error_context.context("Uninstall virtio driver", logging.info)
-        single_driver_install.run(test, params, env)
-        # Need install driver after uninstallation.
-        params["need_uninstall"] = False
-        error_context.context("Install virtio driver", logging.info)
-    else:
+    if params.get("need_uninstall") != "yes":
         error_context.context("Downgrade virtio driver", logging.info)
         change_virtio_media(params["cdrom_virtio_downgrade"])
         single_driver_install.run(test, params, env)
```

Comment:
A minor issue: maybe we should move  line 52:change_virtio_media(params["cdrom_virtio"]) to if part as well.

Reply:
Ah, right, will update it as will, thanks for your review.